yellowbrick/classifier/prcurve.py

```diff
@@ -145,16 +148,18 @@ class PrecisionRecallCurve(ClassificationScoreVisualizer):
     """
 
     def __init__(self, model, ax=None, classes=None, fill_area=True, ap_score=True,
-                 micro=True, iso_f1_curves=False, per_class=False, fill_opacity=0.2,
+                 micro=True, iso_f1_curves=False,f1_values=[0.2,0.4,0.6,0.8], per_class=False, fill_opacity=0.2,
```

Comment:
It is tricky to add mutable objects as default arguments to functions (like Python lists), and so as a general rule of thumb, we try to avoid it. Would you mind making the default value immutable (e.g. either a np.array or a tuple). Some suggestions:

1. `f1_values = np.linspace(0.2, 0.8, num=4)`
2. `f1_values = np.array([0.2, 0.4, 0.6, 0.8])`
3. `f1_values = (0.2, 0.4, 0.6, 0.8)` 

Alternatively, you can create a module variable at the top of the files `DEFAULT_ISO_F1_VALUES` and pass the constant in as this argument.

Reply:
i've changed it to a tuple

Reply:
I'm also raising an exception in case the object passed is mutable ie not a tuple.